plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -63,7 +63,7 @@ function auto_sizes_prime_attachment_caches( $content ): string {
  */
 function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $block ): string {
 	if ( ! is_string( $content ) ) {
-		return '';
+		return ''; // @codeCoverageIgnore
```

Comment:
Ditto. I don't think this should be ignored because technically this is a code path that can be tested.

Reply:
The assertion to test for with `auto_sizes_prime_attachment_caches()` is a bit tricky though. For one, an assertion can assert that the input `$content` is identical to the return value. But then otherwise, when there is a `class` with an attachment ID in it or no `class` attribute at all, then this means `_prime_post_caches()` is called or not. I guess you could check to see if a query was done on the DB or not?

Reply:
```suggestion
		return '';
```